content/events/2018-detroit/program.md

```diff
@@ -0,0 +1,13 @@
++++
+Title = "Program"
+Type = "program"
+Description = "Program for devopsdays detroit 2018"
++++
+
+<div class = "row">
+  <div class = "col">
+    <hr />
+    If Open Space is new to you, you may be interested in <a href="/pages/open-space-format">more details about Open Space</a>.
```

Comment:
`If Open Spaces are new to you`

Reply:
right... this was part of the generated text from the scripts in `utilities`...  maybe we can fix ours and suggest another change separately.

Reply:
Sounds good.